x/eibc/types/fees.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// calculate the new price: transferTotal - fee - bridgingFee
+func CalcPriceWithBridgingFee(amt sdk.Int, feeInt sdk.Int, bridgingFeeMultiplier sdk.Dec) (sdk.Int, error) {
+	bridgingFee := bridgingFeeMultiplier.MulInt(amt).TruncateInt()
+	price := amt.Sub(feeInt).Sub(bridgingFee)
```

Comment:
do we want to protect against bridge fee being zero?

Reply:
it can happen with small amt transfers / low fee percentage

Reply:
is that open to abuse? People can avoid the fee by sending small transfers?
Or maybe it's unrealistic because it would take so long